embeds/questions_embeds.py

```diff
@@ -76,25 +8,28 @@ def daily_problem_unsuccessful_embed() -> discord.Embed:
 
     return embed
 
-
-def question_embed(difficulty: str, question_title: str, rating_text: str, link: str, daily_question: bool = False) -> discord.Embed:
+def question_embed(question_id: str, question_content: str, question_constraints: str, question_difficulty: str, question_title: str, rating_text: str, question_link: str, question_total_accepted, question_total_submission, question_ac_rate, daily_question: bool = False) -> discord.Embed:
```

Comment:
look at my feedback in utils/questions. remove the "question" part of each parameter name